gamemodes/terrortown/gamemode/client/cl_vskin/default_skin.lua

```diff
@@ -1723,6 +1723,7 @@ end
 function SKIN:PaintRoleImageTTT2(panel, w, h)
     local widthBorder = 2
     local widthBorder2 = widthBorder * 2
+    local sizeIconRole = w - widthBorder2
     local padding = 3
     local sizeMode = 18
```

Comment:
Can we make sure this scales with the size of the image as painted?

Reply:
I'm no longer on my PC for today. So yes, we can do this, but I can't do it today.

Also keep in mind that these outlines are hard coded for all UI elements so far

Reply:
The outlines are mostly fine, it's the child icon that poses a problem. See https://github.com/TTT-2/TTT2/pull/1714#issuecomment-2600934166

Reply:
Ah, understood. I will probably have some time in the coming days. Otherwise we merge this as is and you modify it to your liking

Reply:
```suggestion
    local sizeMode = 18 * w / 64
```